netbench-cdk/bin/netbench-cdk.ts

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import { App } from 'aws-cdk-lib';
+import { NetbenchInfra } from "../lib/netbench"
+import { ProdStackPrimaryProps, ProdStackSecondaryProps } from '../lib/config';
+
+const AWS_DEFAULT_REGION = process.env["AWS_DEFAULT_REGION"] || "us-west-2";
+const app = new App();
+
+if (!process.env["DEV_ACCOUNT_ID"]) {
```

Comment:
```suggestion
if (process.env["DEV_ACCOUNT_ID"]) {
```

can you remove the `!`